custom_theme/home.html

```diff
@@ -31,166 +31,161 @@
-                <img class="img-fluid sepia" src="img/home/art02-consensys.svg" alt="ConsenSys" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/consensys-diligence-logo.svg" alt="ConsenSys Diligence" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-5 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art01-microsoft.svg" alt="Microsoft" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art03-ey.svg" alt="Ernst & Young" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art04-airbus.svg" alt="Airbus" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art05-gd.svg" alt="General Dynamics" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art06-amazon.svg" alt="Amazon" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art07-jpmorgan.svg" alt="J.P.Morgan" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid pl-4 pr-4 sepia" src="img/home/art08-vmware.svg" alt="VMWare" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art09-shapeshift.svg" alt="ShapeShift" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid sepia" src="img/home/art10-colony.svg" alt="Colony" />
-            </div>
-            <div class="col-8 col-sm-6 col-md-3 mt-4 mt-sm-3 mb-4 mb-sm-3">
-                <img class="img-fluid pl-4 pr-4 sepia" src="img/home/art11-amberdata.svg" alt="Amberdata" />
-            </div>
+    <div class="row mb-2">
+      <div class="col-md-12 col-sm-12">
+        <div class="card mb-3">
+          <div class="card-header orange">Faster, easier, and safer Ethereum simulation</div>
+          <div class="card-body text-dark">
+            <p class="card-text">Bring up a local blockchain with pre-funded accounts for fast testing with Truffle’s Ganache. Fork mainnet with zero-config, impersonate accounts, auto-mine blocks, and use Ganache programmatically with Node.js. Print variables for quick analysis using console.log and Vyper’s print. Ganache lets you customize a local blockchain for quick testing and mimicking production environments.
```

Comment:
I had difficulty parsing the last sentence. This might be better?

```suggestion
            <p class="card-text">Bring up a local blockchain with pre-funded accounts for fast testing with Truffle’s Ganache. Fork mainnet with zero-config, impersonate accounts, auto-mine blocks, and use Ganache programmatically with Node.js. Print variables for quick analysis using console.log and Vyper’s print. Ganache lets you customize a local blockchain for quick testing and simulation of production environments.
```